terraform/modules/rds/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "allocated_storage" {
+  type        = number
+  description = "The allocated DB storage in gibibytes."
+}
+
+variable "db_subnet_group_name" {
+  type        = string
+  description = "The name of the subnet group associated with the VPC the DB needs to be in."
+}
+
+variable "instance_class" {
+  type        = string
+  description = "The instance class of the DB."
+}
+
+variable "prefix" {
+  type        = string
+  description = "The prefix to be prepended to resource names."
+}
+
+variable "security_group_ids" {

```

Comment:
The description is clear enough, but if you can think of a way of encapsulating it better in this name it'd be worth it - as is it took looking at the description to think this was for allowing access to the db rather than what security group the db should be in.